.github/workflows/windows-msvc.yml

```diff
@@ -0,0 +1,71 @@
+name: Windows MSVC
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  ci-msvc:
+    name: Windows MSVC
+    runs-on: windows-latest
+    env:
+      PKG_CONFIG_PATH: 'C:\gnome\lib\pkgconfig'
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Set up the PATH environment
+      run: |
+        echo "C:\pkg-config-lite-0.28-1\bin" >> $GITHUB_PATH
+        echo "C:\gnome\bin" >> $GITHUB_PATH
+      shell: bash
+        
+    - name: Install pkgconfig-lite
+      run: |
+        Invoke-WebRequest -Uri https://deac-fra.dl.sourceforge.net/project/pkgconfiglite/0.28-1/pkg-config-lite-0.28-1_bin-win32.zip -OutFile /pkg_config_lite.zip
+        Expand-Archive /pkg_config_lite.zip -DestinationPath C:\
+        ls C:\
+        ls C:\pkg-config-lite-0.28-1
+        ls C:\pkg-config-lite-0.28-1\bin
+        pkg-config --version
+
+    - name: Clone GTK
+      run: |
+        cd /
+        git clone https://gitlab.gnome.org/GNOME/gtk.git --depth 1
```

Comment:
As I pointed out earlier this workflow was mainly designed to find regressions in the windows installation instructions. Therefore I think using the latest version is fine and more convenient unless we can update the tag name automatically. Also this would allow us to more quickly find problems with the upstream code I think. 

Or is there a good reason to use a tagged version?

Reply:
Finding regressions sounds good. Let's hope that this build doesn't break too often otherwise it might become annoying.

What I am personally more interested though is testing gtk4-rs om Windows. Here we will want to use the same tagged gtk version that we use in the docker image

Reply:
If we get caching working it should be fine to build gtk in two different workflows.